euphony/src/androidTest/java/co/euphony/rx/EuRxManagerTest.kt

```diff
@@ -9,13 +9,13 @@ class EuRxManagerTest {
 
     @Test(expected = Test.None::class) /* no exception expected */
     fun withExceptionHandlingNoErrorOccurs() {
-        val rxManager = EuRxManager()
+        val rxManager = EuRxManager.getInstance()
         rxManager.listen()
     }
 
     @Test
     fun testListenWithTimeout() {
-        val rxManager = EuRxManager()
+        val rxManager = EuRxManager.getInstance()
         rxManager.listen(1000)
         assertEquals(EuRxManager.RxManagerStatus.RUNNING, rxManager.status)
```

Comment:
This test fail because it is affected by the above test. 😥
Add `rxManager.finish()` to the line 14

Reply:
@zion830 Good catch! 👍 
@phj0407 singleton is really working well!
That's why, 2nd test function will fail as @zion830 mentioned.
so, we need to append `rxManager.finish()` in `withExceptionHandlingNoErrorOccurs()`.
Could you fix this issue?

Reply:
Oh, I see... then, do I fix this issue and make commit and PR again?

Reply:
@phj0407 Yeap! :)